packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -152,44 +172,48 @@ describe('Cloud Logs Processor', () => {
       );
     });
 
+
     describe('correctly formed incoming log messages', () => {
       beforeEach(() => {
         processor = new CloudLogsProcessor({
           firestore: mockFirestore,
           subscription: mockSubscription,
+          listenLimit: LISTEN_LIMIT_SECS,
         });
       });
 
+
       describe('execution start and execution end logs', () => {
         // what the execution record would look like after receiving just the start log
         const executionRecordStart = {
           document: {
             '4ww4alqs7ikq': {
               execution_id: '4ww4alqs7ikq',
-              bot_id: 'merge_on_green',
-              start_time: 1595536893000,
+              bot_name: 'merge_on_green',
+              start_time: 1595536893701,
               logs_url:
-                'https://console.cloud.google.com/logs/query;query=resource.type%3D%22' +
-                'cloud_function%22%0Alabels.%22execution_id%22%3D%224ww4alqs7ikq%22;' +
-                'timeRange=2020-07-23T20:41:33.701320846Z%2F22020-07-23T20:41:33.701320846Z;' +
-                'summaryFields=:true:32:beginning?project=repo-automation-bots',
+                'https://pantheon.corp.google.com/logs/query;query=labels.execution_id' +
```

Comment:
The url is more simplified now.

Also I had to change `console.cloud.google.com` back to `pantheon.corp.google.com` because the redirect from `console.cloud` -> `pantheon.corp` erases the query parameters and the link doesn't work. 

I'll add a `TODO` to figure out a solution in the implementation.